Service/ContentTypeService.php

```diff
@@ -426,10 +430,30 @@ public function getAllTypes()
         return implode(',', array_keys($this->contentTypeArrayByName));
     }
 
-    public function contentTypeFromJson(string $json, Environment $environment): ContentType
+
+    public function updateFromJson(ContentType $contentType, string $json, bool $isDeleteExitingTemplates, bool $isDeleteExitingViews): void
```

Comment:
the analogy with contentTypeFromJson is distracting because it indicates the creation of a ContentType, from a Json input. updateFromJson indicates the creation of Update from Json input which is incorrect (maybe just "updateContentType"? As the function is updating the contentType passed as argument).


Also small fixes in naming of the parameters.